src/EA.Weee.Web.Tests.Unit/Areas/Aatf/Controller/ManageEvidenceNotesControllerIndexTests.cs

```diff
@@ -345,13 +266,8 @@ public async void IndexGetWithDefaultTab_GivenRequiredDataAndFilterModel_ModelMa
                      && m.AatfData.Equals(aatfData)
                      && m.OrganisationId.Equals(organisationId)
                      && m.FilterViewModel.Equals(filter.FilterViewModel)
-                     && m.CurrentDate == currentDate))).MustHaveHappenedOnceExactly();
-
-            foreach (var aatf in selectYourAatfViewModel.AatfList)
-            {
-                A.CallTo(() => Mapper.Map<ManageEvidenceNoteViewModel>(A<ManageEvidenceNoteTransfer>.That.Matches(
-                    m => m.Aatfs.Contains(aatf)))).MustHaveHappenedOnceExactly();
-            }
+                     && m.CurrentDate == currentDate
+                     && m.Aatfs.SequenceEqual(aatfs)))).MustHaveHappenedOnceExactly();
         }
 
```

Comment:
Are there more unit tests to come for the controller? Mostly thinking centered around the annotation/attribute changes of so many methods